src/module/item/helpers.ts

```diff
@@ -74,4 +82,91 @@ function markdownToHTML(markdown: string): string {
         .innerHTML.trim();
 }
 
-export { itemIsOfType, markdownToHTML, performLatePreparation, reduceItemName };
+const valueRegex = /^([\w-]+)-(\d{1,3})$/;
+
+function evaluateTraitData(traits?: string[]): ValuedTraits {
```

Comment:
```suggestion
function evaluateTraitConfiguration(traits?: string[]): ValuedTraits {
```
Or `evaluateTraitConfig`?